ko/news/_posts/2024-08-22-dos-rexml-cve-2024-43398.md

```diff
@@ -1,31 +1,31 @@
 ---
 layout: news_post
-title: "CVE-2024-43398: DoS vulnerability in REXML"
+title: "CVE-2024-43398: REXML의 DoS 취약점"
 author: "kou"
-translator:
+translator: "shia"
 date: 2024-08-22 03:00:00 +0000
 tags: security
-lang: en
+lang: ko
 ---
 
-There is a DoS vulnerability in REXML gem. This vulnerability has been assigned the CVE identifier [CVE-2024-43398](https://www.cve.org/CVERecord?id=CVE-2024-43398). We strongly recommend upgrading the REXML gem.
+REXML gem에서 DoS 취약점이 발견되었습니다. 이 취약점은 CVE 번호 [CVE-2024-43398](https://www.cve.org/CVERecord?id=CVE-2024-43398)로 등록되었습니다. REXML gem 업그레이드를 강하게 추천합니다.
 
-## Details
+## 세부 내용
 
-When parsing an XML that has many deep elements that have same local name attributes.
+동일한 지역 이름 속성을 가진 여러 깊은 요소를 포함하는 XML을 파싱할 때, REXML gem은 처리에 긴 시간이 걸립니다.
```

Comment:
Origin text doesn't include "may take long time", but It's so weird without this in korean, imo, ;)

Reply:
I left a comment. https://github.com/ruby/www.ruby-lang.org/pull/3348#discussion_r1731126945